src/Integration/Telemetry/TelemetryPayload.cs

```diff
@@ -40,6 +40,9 @@ public sealed class TelemetryPayload
         [JsonProperty("ide_version")]
         public string VisualStudioVersion { get; set; }
 
+        [JsonProperty("ide_version_information")]
```

Comment:
After discussion with Julien, we decided against commonly named fields -- the reasoning is that each IDE has its own version concepts, and they already had situations in the past where they wanted to know more intellij-specific information that doesn't fit into these 3 fields. So there is no point in trying to come up with a common schema.

Hence, `ide_version_information` might be better named as `slvs_ide_version`. I haven't changed it yet, please give your feedback on the PR and the names and I'll do the changes in bulk.

Reply:
`slvs_` would stop the data being accidentally misinterpreted when looking at telemetry. In fact, the "sl" bit is redundant, so just `vs_version_information` would be enough.

Reply:
Julien suggested that it be named "slvs_", as a way to filter out flavor-specific properties. Just "vs_" won't be as eye-catching to the rest of the team.